src/index.tsx

```
@@ -155,6 +163,10 @@ const InternalSegmentedOption: React.FC<{
       </div>
     </label>
   );
+
+  const renderProps = { ...props };
+  delete renderProps.itemRender;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)